doc/api/fs.md

```diff
@@ -6909,6 +6909,89 @@ changes:
 
 The path to the parent directory of the file this {fs.Dirent} object refers to.
 
+### Class: `fs.FastUtf8Stream`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An optimized UTF-8 stream writer.
```

Comment:
I think this needs a bit more explanation, specifically around flushing and buffering, as a normal write stream could lose data.

Reply:
Yeah, I'll be expanding the documentation before moving this PR out of draft. But feel free to offer suggestions ;-)

Reply:
```suggestion
An optimized UTF-8 stream writer that allows for flushing all the internal buffering on demand. It handles EAGAIN errors correctly, allowing for customization, for example, by dropping content if the disk is busy. 
```